src/App.jsx

```diff
@@ -1,10 +1,76 @@
-import React from 'react';
+import React, { Component } from "react";
+import axios from "axios";
 
-function App() {
-  return (
-    <div>
-    </div>
-  );
+class App extends Component {
+  state = {
+    query: "",
+    results: [],
+    message: ""
+  };
+
+  onSubmitFormHandler = async e => {
+    e.preventDefault();
+    let response = await axios.post("https://cocktails-api-team1.herokuapp.com/api/v1/cocktails", {
+      params: {
+        query: "name"
+      }
+    });
+    if (response.data.status === 400) {
+      debugger;
+      this.setState({
+        message: response.data.message,
+        results: null
+      });
+    } else {
+      debugger;
+      this.setState({
+        results: response.data.drinks
+      });
+    }
+  };
+
+  render() {
+    let renderResults;
+    if (Array.isArray(this.state.results) && this.state.results.length > 0) {
+      debugger;
+      renderResults = (
+        <div id="result-list">
+          {this.state.results.map(item => {
+            debugger;
+            return (
+              <div key={item.idDrink}>
+                {item.strDrink}
+                {item.name}
+                {item.strCategory} {item.strIBA}
+              </div>
+            );
+          })}
+        </div>
+      );
+    } else {
+      renderResults = <div id="message">{this.state.message}</div>;
+    }
+
+    return (
+      <>
+        <form id="search-by-name">
+          <div className="ui search">
+            <input
+              name="search-by-name"
+              type="text"
+              id="name-search"
+              className="prompt"
+              placeholder="Search by drink name"
+            ></input>
+            <button id="search" onClick={this.onSubmitFormHandler}>
```

Comment:
If you have a `onClick` event on this button, why are you even using the `form` tag?  Keep it simple ppl.

Reply:
We have switched to a formHandler rather than onClick for consistency for using the form. We understand that we could either the formHandler or onClick on the button.